apps/mobile/src/libs/components/wheel/libs/components/animated-appearing-sector/animated-appearing-sector.tsx

```diff
@@ -0,0 +1,86 @@
+import { Animated, Circle } from "~/libs/components/components";
+import { type SectorInitialSharedValues } from "~/libs/types/types";
+
+import { useWheelAnimation } from "../../hooks/hooks";
+
+type Properties = {
+	animationRepetitions: number;
+	animationTime: number;
+	centerGap: number;
+	centerPoint: number;
+	endPercentInner: number;
+	endPercentOuter: number;
+	height: number;
+	layerOffset: number;
+	outlineColor: string;
+	sectorColor: string;
+	startPercentInner: number;
+	startPercentOuter: number;
+};
+
+const INITIAL_ANIMATED_VALUE = 0;
+const AnimatedCircle = Animated.createAnimatedComponent(Circle);
+
+const AnimatedAppearingSector: React.FC<Properties> = ({
+	animationRepetitions,
+	animationTime,
+	centerGap,
+	centerPoint,
+	endPercentInner,
+	endPercentOuter,
+	height,
+	layerOffset,
+	outlineColor,
+	sectorColor,
+	startPercentInner,
+	startPercentOuter,
+}: Properties) => {
+	const initialSharedValues: SectorInitialSharedValues = {
```

Comment:
```suggestion
	const INITIAL_SHARED_DATA: SectorInitialSharedValues = {
```
And move it to a separate file